// @generated SignedSource<<873b06e7854355e4c3d9e8b72b0cf8ce>>
// DO NOT EDIT THIS FILE MANUALLY!
// This file is a mechanical copy of the version in the configerator repo. To
// modify it, edit the copy in the configerator repo instead and copy it over by
// running the following in your fbcode directory:
//
// configerator-thrift-updater glean/service.thrift

namespace cpp2 facebook.glean
namespace hs Glean
namespace py glean.service
namespace py3 glean
namespace php glean
namespace rust glean_service

struct HostPort {
  1: string host
  2: i32 port
}

union Service {
  1: string tier
  2: HostPort hostPort
} (hs.nonempty)
